fprj-core/src/net/fproject/utils/DateTimeUtil.as

```diff
@@ -774,6 +776,15 @@ package net.fproject.utils
 			return sb;
 		}
 		
+		public static function formatToUniversalDate(d:Date, includeMilliseconds:Boolean=false,
```

Comment:
Anh thấy đặt tên như này chưa hợp lý. Nên đổi thành formatUTCDate(...) vì Flex framework hay sử dụng từ UTC, và phải bỏ giới từ "To" đi